### PR TITLE
Perform bytes to string and vice versa conversions

### PR DESCRIPTION
**Pull request by candrews**
Python 3 handles bytes and strings as different types

Fixes https://github.com/ehloonion/onionrouter/issues/19

### CODE REVIEW

**Comment by ioparaskev** [comment]
Yes indeed this seems like a python3 bug

File: onionrouter/msockets.py [hunk 1]
```diff
@@ -22,12 +22,12 @@ def resolve(rerouter, conn, resolve_callback=lambda q, a: (q, a)):
...
                 result = rerouter.run(addr)
                 resolve_callback(addr, result)
-                conn.sendall("{0}\n".format(result))
+                conn.sendall("{0}\n".format(result).encode())
     except socket.timeout:
         return
     except BaseException as err:
         # todo log
-        conn.sendall("500 {0}\n".format(err))
+        conn.sendall("500 {0}".format(err).encode())
```
Comment (ioparaskev) [inline onionrouter/msockets.py:30]:
```suggestion
        conn.sendall("500 {0}\n".format(err).encode())
```

**Comment by candrews** [comment]
Could please add the `hacktoberfest-accepted` tag to this PR?

**Comment by ioparaskev** [comment]
I guess you mean PR label? Just added it

**Comment by candrews** [comment]
Thank you once again! :tada:

**Pull request merged**